docs/src/modules/java/pages/kickstart.adoc

```diff
@@ -133,7 +133,7 @@ For more details see xref:java:value-entity.adoc[].
 
 === Unit and integration tests
 
-The Akka Serverless Maven plugin creates a unit test stub for each class. Use this stub as a starting point to test the logic in your implementation. The Akka Serverless Java SDK test kit supports both link:https://junit.org/junit4/[JUnit 4] and link:https://junit.org/junit5/[JUnit 5].
+The Akka Serverless Maven plugin creates a unit test stub for Entity. Use this stub as a starting point to test the logic in your implementation. The Akka Serverless Java SDK test kit supports both link:https://junit.org/junit4/[JUnit 4] and link:https://junit.org/junit5/[JUnit 5].
```

Comment:
```suggestion
The Akka Serverless Maven plugin creates a unit test stub for the Entity. Use this stub as a starting point to test the logic in your implementation. The Akka Serverless Java SDK test kit supports both link:https://junit.org/junit4/[JUnit 4] and link:https://junit.org/junit5/[JUnit 5].
```

Reply:
We once decided to stick with junit5. 
See https://github.com/lightbend/akkaserverless-java-sdk/issues/41

I don't remember from where the idea comes from 'supporting'. Is it possible that the codegen was previously generating something for one or other style of tests? 

Anyway, I don't think we should change that in this PR, but we probably want to address #41 before we release a new SDK.